pkg/eks/services_v2.go

```diff
@@ -43,6 +45,22 @@ func (s *ServicesV2) STSV2() awsapi.STS {
 	return s.sts
 }
 
+// STSV2Presign provides a signed STS client for calls to Kubernetes.
+func (s *ServicesV2) STSV2Presign() v1alpha5.STSPresigner {
```

Comment:
Based on the above suggestion:
```suggestion
func (s *ServicesV2) STSV2Presign() api.STSPresigner {
```